cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/KafkaCruiseControlConfig.java

```diff
@@ -329,6 +329,14 @@
       + "replica distribution from each topic. For example, 1.80 means the highest topic replica count of a broker "
       + "should not be above 1.80x of average replica count of all brokers for the same topic.";
 
+  /**
+   * <code>goal.violation.distribution.threshold.multiplier</code>
+   */
+  public static final String GOAL_VIOLATION_DISTRIBUTION_THRESHOLD_MULTIPLIER_CONFIG = "goal.violation.distribution.threshold.multiplier";
+  private static final String GOAL_VIOLATION_DISTRIBUTION_THRESHOLD_MULTIPLIER_DOC = "The multiplier applied to the threshold"
+      + " of distribution goals used for detecting and fixing anomalies. For example, 2.50 means the threshold for each "
```

Comment:
Should we mention that `LeaderBytesInDistributionGoal` is not relevant here?

Reply:
Updated with relevant goals.